bootstrap/eks/controllers/eksconfig_controller_reconciler_tests.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	bootstrapv1 "sigs.k8s.io/cluster-api-provider-aws/bootstrap/eks/api/v1alpha3"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/test/helpers"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	testEnv *helpers.TestEnvironment
```

Comment:
`testEnv` is already declared in: https://github.com/newrelic-forks/cluster-api-provider-aws/blob/93384b83e9def30779ea4940675f4997cb3b74bf/bootstrap/eks/controllers/suite_test.go#L42
I am think: remove this one and change the one in `suite_test.go` to look like `testEnv *helpers.TestEnvironment` (same approach as in [kubeadm](https://github.com/kubernetes-sigs/cluster-api/blob/4e07b0c23f6782fa5ca3729c3f9ed005a6f9a203/controlplane/kubeadm/controllers/suite_test.go#L34))

Reply:
😮 I didn't notice it in `suite_test.go`
Will resolve this. Interesting that running the test cases for me did not call out the redeclaration 🤔 

Thanks @h0tbird !!!